### PR TITLE
Created a directory for Quality Council

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Created a directory for Quality Council : it's a start before adding a lot of our processes
| Fixed ticket | N/A

FYI : @AureRita / @Robin-Fischer-PS / @boubkerbribri / @florine2623

### CODE REVIEW

**Comment by Progi1984** [comment]
Ping @kpodemski / @coffrep for review.

**Comment by kpodemski** [comment]
thanks, @Progi1984, but we're going to restructure this page anyway, we haven't discussed yet how, but there will be information about council members, etc.

**Comment by Progi1984** [comment]
@kpodemski I have created a new space for the Quality Council with its own process, and so on... (soon)

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+# 2023-05-16
+
+## Presents
+
+* [@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS) : Member of the Quality Coucil
+* [@boubkerbribri](https://github.com/boubkerbribri) : Member of the Quality Coucil
+* [@AureRita](https://github.com/AureRita)
+* [@florine2623](https://github.com/AureRita)
```
Comment (PrestaEdit) [inline content/quality-council/meetings/2023-05-16-meeting.md:12]: There is a mistake, here :)

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* [@Progi1984](https://github.com/Progi1984)
+
+## Report
+
+Like Pablo Borowicz said, each council can organize like it wants. So this is the goal of this meeting : define rules for the Quality Coucil.
```
Comment (PrestaEdit) [inline content/quality-council/meetings/2023-05-16-meeting.md:17]: Council, not Coucil.

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+### Members of the council
+
+There are 3 profiles which are members of the Quality Coucil :
```
Comment (PrestaEdit) [inline content/quality-council/meetings/2023-05-16-meeting.md:21]: Council, bis

File: content/quality-council/meetings/2023-05-16-meeting.md (hunk 2 above)
Comment (Progi1984) [reply content/quality-council/meetings/2023-05-16-meeting.md:17]: Fixed

File: content/quality-council/meetings/2023-05-16-meeting.md (hunk 3 above)
Comment (Progi1984) [reply content/quality-council/meetings/2023-05-16-meeting.md:21]: Fixed all

File: content/quality-council/rules/_index.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+### Members
+
+There are 3 profiles which are members of the Quality Coucil :
+* Issue Manager
+* Software Developer in Test
+* Software Tester
+
+Each member can be : 
+* either a maintainer (can participate and vote during the Quality Coucil),
+* or a committer (can participate during the Quality Coucil).
```
Comment (PrestaEdit) [inline content/quality-council/rules/_index.md:16]: Council, again

File: content/quality-council/rules/_index.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+### Members
+
+There are 3 profiles which are members of the Quality Coucil :
+* Issue Manager
+* Software Developer in Test
+* Software Tester
+
+Each member can be : 
+* either a maintainer (can participate and vote during the Quality Coucil),
```
Comment (PrestaEdit) [inline content/quality-council/rules/_index.md:15]: Council, again

File: content/quality-council/rules/_index.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+---
+title: Rules of the Council
+---
+
+##  Rules of the Council
+
+### Members
+
+There are 3 profiles which are members of the Quality Coucil :
```
Comment (PrestaEdit) [inline content/quality-council/rules/_index.md:9]: Council, again

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+* Only maintainers can vote
+* Each maintainer has one vote
+* Results of the vote are public
+* Vote is accepted when a simple majority is reached
+
+### Lead Council
+
+* Actual Lead is [@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS)
+* He is elected for one year (until 1st of April 2024) by maintainers.
```
Comment (kpodemski) [inline content/quality-council/meetings/2023-05-16-meeting.md:42]: Leads are not elected by maintainers but chosen by Leadership council.

File: content/quality-council/rules/_index.md (hunk 5 above)
Comment (Progi1984) [reply content/quality-council/rules/_index.md:15]: Fixed

File: content/quality-council/rules/_index.md (hunk 4 above)
Comment (Progi1984) [reply content/quality-council/rules/_index.md:16]: Fixed

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+### Members of the council
+
+There are 3 profiles which are members of the Quality Coucil :
+* Issue Manager
+* Software Developer in Test
+* Software Tester
+
+Each member can be : 
+* either a maintainer (can participate and vote during the Quality Coucil),
```
Comment (PrestaEdit) [inline content/quality-council/meetings/2023-05-16-meeting.md:27]: Council, again

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
+---
+title: 2023-05-16
+---
+
+# 2023-05-16
+
+## Presents
+
+* [@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS) : Member of the Quality Coucil
```
Comment (PrestaEdit) [inline content/quality-council/meetings/2023-05-16-meeting.md:9]: Council, again

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+* [@boubkerbribri](https://github.com/boubkerbribri) : Member of the Quality Coucil
```
Comment (PrestaEdit) [inline content/quality-council/meetings/2023-05-16-meeting.md:10]: Council, again

File: content/quality-council/meetings/2023-05-16-meeting.md (hunk 7 above)
Comment (Progi1984) [reply content/quality-council/meetings/2023-05-16-meeting.md:42]: I disagree with that : 

> For the time being, the project’s main sponsor, PrestaShop SA, reserves the right to appoint the Leadership Council’s lead.

Source : https://build.prestashop-project.org/news/2023/a-new-organization-for-the-open-source-project/

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+### Members of the council
+
+There are 3 profiles which are members of the Quality Coucil :
+* Issue Manager
+* Software Developer in Test
+* Software Tester
+
+Each member can be : 
+* either a maintainer (can participate and vote during the Quality Coucil),
+* or a committer (can participate during the Quality Coucil).
```
[the remark this answers is not in the document]
Comment (Progi1984) [reply content/quality-council/meetings/2023-05-16-meeting.md:28]: Fixed

File: content/quality-council/meetings/2023-05-16-meeting.md (hunk 8 above)
Comment (Progi1984) [reply content/quality-council/meetings/2023-05-16-meeting.md:27]: Fixed

File: content/quality-council/meetings/2023-05-16-meeting.md (hunk 9 above)
Comment (Progi1984) [reply content/quality-council/meetings/2023-05-16-meeting.md:9]: Fixed

File: content/quality-council/meetings/2023-05-16-meeting.md (hunk 10 above)
Comment (Progi1984) [reply content/quality-council/meetings/2023-05-16-meeting.md:10]: Fixed

**Review by Robin-Fischer-PS — APPROVED** [review]
OK for me :+1:

**Review by boubkerbribri — APPROVED** [review]
LGTM

File: content/get-involved/_index.md [hunk 12]
```diff
@@ -28,8 +28,8 @@ More information: [How to contribute (DevDocs)][how-to-contribute]
 
 [report-bug]: {{< relref "report-issues" >}}
 [request-feature]: {{< relref "request-features" >}}
-[review-code]: {{< relref "/maintainers-guide/processes/reviewing-pull-requests.md" >}}
-[test-code]: {{< relref "/maintainers-guide/processes/test-pr-and-branches/" >}}
+[review-code]: {{< relref "/quality-council/processes/reviewing-pull-requests.md" >}}
```
Comment (kpodemski) [inline content/get-involved/_index.md:31]: @Progi1984 don't you think it targets more tech committers/maintainers? 🤔

File: content/project-organization/people-and-roles.md [hunk 13]
```diff
@@ -69,51 +69,58 @@ Current UX designers:
...
+* Maintainers :
+  - Aurélien Rita ([@AureRita](https://github.com/AureRita))
+  - Faten Mhiri ([@MhiriFaten](https://github.com/MhiriFaten))
+  - Florine Hea ([@florine2623](https://github.com/florine2623))
+  - Hajer Elleuch ([@djoelleuch](https://github.com/djoelleuch))
+  - Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
+  - Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
+  - Ines Sallemi ([@sallemiines](https://github.com/sallemiines))
+  - Paul-Noël Cholot ([@paulnoelcholot](https://github.com/paulnoelcholot))
+* Committers :
```
Comment (kpodemski) [inline content/project-organization/people-and-roles.md:88]:
```suggestion
* Committers: 
```

File: content/project-organization/people-and-roles.md [hunk 14]
```diff
@@ -69,51 +69,58 @@ Current UX designers:
...
-- Hajer Elleuch ([@djoelleuch](https://github.com/djoelleuch))
-- Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
-- Ines Sallemi ([@sallemiines](https://github.com/sallemiines))
-- Khouloud Belguith ([@khouloudbelguith](https://github.com/khouloudbelguith))
-- Paul-Noël Cholot ([@paulnoelcholot](https://github.com/paulnoelcholot))
-- Robin Fischer ([@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS))
-- Sarah Dib ([@sarahdib](https://github.com/sarahdib))
-
-### Software Developers in Test
+* Maintainers :
```
Comment (kpodemski) [inline content/project-organization/people-and-roles.md:97]:
```suggestion
* Maintainers:
```

File: content/project-organization/people-and-roles.md [hunk 15]
```diff
@@ -69,51 +69,58 @@ Current UX designers:
...
+  - Aurélien Rita ([@AureRita](https://github.com/AureRita))
+  - Faten Mhiri ([@MhiriFaten](https://github.com/MhiriFaten))
+  - Florine Hea ([@florine2623](https://github.com/florine2623))
+  - Hajer Elleuch ([@djoelleuch](https://github.com/djoelleuch))
+  - Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
+  - Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
+  - Ines Sallemi ([@sallemiines](https://github.com/sallemiines))
+  - Paul-Noël Cholot ([@paulnoelcholot](https://github.com/paulnoelcholot))
+  - Robin-Fischer-PS ([@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS))
+* Committers :
```
Comment (kpodemski) [inline content/project-organization/people-and-roles.md:107]:
```suggestion
* Committers: 
```

File: content/project-organization/people-and-roles.md [hunk 16]
```diff
@@ -69,51 +69,58 @@ Current UX designers:
...
 Software Developers in Test provide technical vision, review contribution and write automated tests, which help accelerate the testing process.
 
 Current Software Developers in Test:
 
-- Boubker Bribri ([@boubkerbribri](https://github.com/boubkerbribri))
-- Chedly Farhani ([@cfarhani06](https://github.com/cfarhani06))
-- Franck Lefèvre ([@Progi1984](https://github.com/Progi1984))
-- Nesrine Abdmouleh ([@nesrineabdmouleh](https://github.com/nesrineabdmouleh))
-- Stéphane Decisy ([@SD1982](https://github.com/SD1982))
+* Maintainers :
```
Comment (kpodemski) [inline content/project-organization/people-and-roles.md:117]:
```suggestion
* Maintainers:
```

File: content/project-organization/people-and-roles.md [hunk 17]
```diff
@@ -69,51 +69,58 @@ Current UX designers:
...
+  - Boubker Bribri ([@boubkerbribri](https://github.com/boubkerbribri))
+  - Nesrine Abdmouleh ([@nesrineabdmouleh](https://github.com/nesrineabdmouleh))
+* Committers :
```
Comment (kpodemski) [inline content/project-organization/people-and-roles.md:120]:
```suggestion
* Committers: 
```

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 18]
```diff
@@ -0,0 +1,46 @@
...
+* [@florine2623](https://github.com/florine2623)
+* [@Progi1984](https://github.com/Progi1984)
+
+## Report
+
+Like Pablo Borowicz said, each council can organize like it wants. So this is the goal of this meeting : define rules for the Quality Council.
+
+### Members of the council
+
+There are 3 profiles which are members of the Quality Council :
```
Comment (kpodemski) [inline content/quality-council/meetings/2023-05-16-meeting.md:21]:
```suggestion
There are 3 profiles which are members of the Quality Council:
```

File: content/quality-council/meetings/2023-05-16-meeting.md [hunk 19]
```diff
@@ -0,0 +1,46 @@
...
+* Issue Manager
+* Software Developer in Test
+* Software Tester
+
+Each member can be :
```
Comment (kpodemski) [inline content/quality-council/meetings/2023-05-16-meeting.md:26]:
```suggestion
Each member can be: 
```

File: content/quality-council/rules/_index.md [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+##  Rules of the Council
+
+### Members
+
+There are 3 profiles which are members of the Quality Council :
+* Issue Manager
+* Software Developer in Test
+* Software Tester
+
+Each member can be :
```
Comment (kpodemski) [inline content/quality-council/rules/_index.md:14]:
```suggestion
Each member can be: 
```

File: content/get-involved/_index.md (hunk 12 above)
Comment (Progi1984) [reply content/get-involved/_index.md:31]: Re-moved

File: content/project-organization/people-and-roles.md [hunk 21]
```diff
@@ -69,51 +69,58 @@ Current UX designers:
...
-- Aurélien Rita ([@AureRita](https://github.com/AureRita))
-- Faten Mhiri ([@MhiriFaten](https://github.com/MhiriFaten))
-- Florine Hea ([@florine2623](https://github.com/florine2623))
-- Hajer Elleuch ([@djoelleuch](https://github.com/djoelleuch))
-- Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
-- Hibatallah Aouadni ([@hibatallahAouadni](https://github.com/hibatallahAouadni))
-- Ines Sallemi ([@sallemiines](https://github.com/sallemiines))
-- Khouloud Belguith ([@khouloudbelguith](https://github.com/khouloudbelguith))
-- Paul-Noël Cholot ([@paulnoelcholot](https://github.com/paulnoelcholot))
+* Maintainers :
```
[the remark this answers is not in the document]
Comment (Progi1984) [reply content/project-organization/people-and-roles.md:79]: Fixed

File: content/project-organization/people-and-roles.md (hunk 14 above)
Comment (Progi1984) [reply content/project-organization/people-and-roles.md:97]: Fixed

File: content/project-organization/people-and-roles.md (hunk 16 above)
Comment (Progi1984) [reply content/project-organization/people-and-roles.md:117]: Fixed

File: content/project-organization/people-and-roles.md (hunk 15 above)
Comment (Progi1984) [reply content/project-organization/people-and-roles.md:107]: Fixed

File: content/project-organization/people-and-roles.md (hunk 17 above)
Comment (Progi1984) [reply content/project-organization/people-and-roles.md:120]: Fixed

File: content/quality-council/meetings/2023-05-16-meeting.md (hunk 18 above)
Comment (Progi1984) [reply content/quality-council/meetings/2023-05-16-meeting.md:21]: Fixed

File: content/quality-council/meetings/2023-05-16-meeting.md (hunk 19 above)
Comment (Progi1984) [reply content/quality-council/meetings/2023-05-16-meeting.md:26]: Fixed

File: content/quality-council/rules/_index.md (hunk 20 above)
Comment (Progi1984) [reply content/quality-council/rules/_index.md:14]: Fixed

File: content/project-organization/people-and-roles.md [hunk 22]
```diff
@@ -69,51 +69,58 @@ Current UX designers:
...
+  - Aurélien Rita ([@AureRita](https://github.com/AureRita))
+  - Faten Mhiri ([@MhiriFaten](https://github.com/MhiriFaten))
+  - Florine Hea ([@florine2623](https://github.com/florine2623))
+  - Hajer Elleuch ([@djoelleuch](https://github.com/djoelleuch))
+  - Hana Rebai ([@HanaRebaiQA](https://github.com/HanaRebaiQA))
```
[the remark this answers is not in the document]
Comment (Progi1984) [reply content/project-organization/people-and-roles.md:84]: I just confirmed with @Robin-Fischer-PS. @HanaRebaiQA remains as an external maintainer for the moment.

**Comment by Progi1984** [comment]
@kpodemski Fixed feedbacks

**Review by kpodemski — APPROVED** [review]

**Pull request merged**